packages/web/src/components/track/mobile/ConnectedTrackTile.tsx

```diff
@@ -242,9 +244,10 @@ const ConnectedTrackTile = ({
       // Playback
       permalink={permalink}
       togglePlay={togglePlay}
-      isActive={uid === playingUid}
+      isActive={uid === playingUid || isActive}
       isLoading={loading}
-      isPlaying={uid === playingUid && isPlaying}
+      isPlaying={uid === playingUid || isPlaying}
```

Comment:
@dylanjeffers i see you last changed this to include the && isPlaying. let's sync on that when you're available so i can avoid breaking sthg

Reply:
nvm i saw the bug. reverting this line